projects/observability/src/pages/explorer/explorer-service.test.ts

```diff
@@ -113,4 +138,28 @@ describe('Explorer service', () => {
       });
     });
   });
+
+  test('creates external nav params correctly when timeRange is provided', () => {
+    runFakeRxjs(({ expectObservable }) => {
+      const spectator = createService();
+      expectObservable(
+        spectator.service.buildExternalNavParamsWithFilters(
+          ScopeQueryParam.EndpointTraces,
+          [
+            { field: 'duration', operator: FilterOperator.GreaterThan, value: 200 },
+            { field: 'status', operator: FilterOperator.Equals, value: 404 }
+          ],
+          mockTimeRange
+        )
+      ).toBe('(x|)', {
+        x: {
+          navType: NavigationParamsType.External,
+          url:
+            '/explorer?filter=duration_gt_200&filter=status_eq_404&scope=endpoint-traces' +
+            `&time=${mockTimeRange.toUrlString()}`,
+          windowHandling: ExternalNavigationWindowHandling.NewWindow
```

Comment:
We just need the `windowHandling`. No need to set the nav type to External

Reply:
We could use `OpenInNewTabModule`

Reply:
Oh yeah I forgot we had that. We should update it to just take in nav params or string and have the component handle any externalization necessary